### ISSUE TITLE
Requires private "login" gem

### ISSUE THREAD

**Issue by daveshenk** [stakach/em_control#1]
bundle install requests the password for git@advanced.plan.io:advanced-authenticator.git

**Comment by stakach** [comment]
Yeah, we are going to develop the system further before releasing the project officially.
I need to document, provide tests and write a few tutorials.

Do you have a project in mind? I'd be happy to write up some instructions on writing modules and interfaces.
Ill make the login system public, however there is still an issue with rails admin overwriting the environment task which requires a minor customisation.

**Comment by daveshenk** [comment]
I have a networked Integra AV Receiver I've been playing around with, I've been able to talk to it with TCPSocket.open, but it doesn't like me opening and closing connections so often. I've also got one of those iTach IR emitters I'd like to use to control some other devices, and a networked DirecTV box. EventMachine seems like the perfect solution to this little problem, and this repo seems to be doing exactly what I had in mind.

I tried commenting out # before_filter :auth_user but it's still giving me grief about not knowing what to do with Login.
